.github/workflows/ci.yml

```diff
@@ -174,6 +174,20 @@ jobs:
       with:
         name: simplecov-resultset-rails${{matrix.rails_version}}-ruby${{matrix.ruby_version}}
         path: coverage
+  deprecated:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 3.0.x
+    - name: Make sure deprecated components can be linted
+      run: |
+        gem install bundler:2.2.9
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
+        bundle exec rake deprecated:check
```

Comment:
should I move this check elsewhere?

would this make more sense in `.github/workflows/docs-build.yml` after the `bundle exec rake static:dump`? 

last I remember the docs build action was funky so wasn't sure

Reply:
I think it makes sense here since it doesn't really have to do with docs. What if we added `bundle exec rake static:dump` before the `rake deprecated:check` here? It's doing the work again but should be fast enough.

Reply:
Oooh makes sense to me 👍